homework_04/05_viktoriya_vasileva/homework_04_05_viktoriya_vasileva.yaml

```diff
@@ -0,0 +1,34 @@
+name: Pylint
+
+on:
+    push:
+        branches: [ main ]
+        paths: 
+          - homework_04/05_viktoriya_vasileva/**
+    pull_request:
+        branches: [ main ]
+        paths: 
+          - homework_04/05_viktoriya_vasileva/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9"]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+    - name: Analysing the code with pylint
+      run: |
+        cd homework_04
+        cd 05_viktoriya_vasileva
+        pylint circles.py
+        pylint test_circles.py
```

Comment:
This file needs to be in .github/workflows so the checks actually run. That way github does not recognise the yaml file as a workflow so it does not run neither on push nor pull request.

Reply:
You also do not have a "run" which runs the tests. You need to add that as well.

Reply:
+1